### PR TITLE
Css modules

### PR DESCRIPTION
**Pull request by mflinn92**
Implemented some basic styling to match the look of etsy using css modules.

### CODE REVIEW

File: client/components/onOrderAvailable.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import React from 'react';
 const OnOrderAvailable = ({availableQuantity, onOrder}) => {
   let popularItem = (
     <div className="popularItem">
-      <em>Other people want this.</em> {onOrder} people have this in their carts right now.
+      <b>Other people want this.</b> {onOrder} people have this in their carts right now.
```
Comment (therobinkim) [inline client/components/onOrderAvailable.js]: `<b>` tags are deprecated :(

Comment (therobinkim) [reply client/components/onOrderAvailable.js]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

File: public/bundle.js [hunk 2]
```diff
@@ -94,7 +94,7 @@
...
...zdGF0ZS5vbk9yZGVyfSAvPlxyXG4gICAgICAgIDxociBjbGFzc05hbWU9e3N0eWxlcy5ib3JkZXJMaW5lfSAvPlxyXG4gICAgICA8L2Rpdj5cclxuICAgIClcclxuICB9XHJcbn1cclxuXHJcbmV4cG9ydCBkZWZhdWx0IEFwcDtcclxuIl0sIm1hcHBpbmdzIjoiOzs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7OztBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7Ozs7O0FBRUE7QUFBQTtBQUNBO0FBREE7QUFDQTtBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFmQTtBQWlCQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBdkJBO0FBd0JBO0FBQ0E7Ozs7OztBQUNBOzs7Ozs7OztBQUVBO0FBQ0E7O0FBREE7QUFDQTtBQUNBOzs7O0FBQ0E7QUFDQTs7Ozs7Ozs7Ozs7Ozs7Ozs7QUFHQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUZBO0FBSUE7OztBQUVBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFDQTs7O0FBRUE7QUFBQTtBQUNBO0FBQUE7QUFDQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFDQTtBQUFBO0FBRUE7QUFDQTtBQUNBO0FBSEE7QUFLQTtBQUFBO0FBQ0E7QUFBQTtBQUNBO0FBQUE7QUFDQTtBQUFBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7QUFBQTtBQUNBO0FBQUE7QUFBQTtBQUVBO0FBQUE7QUFBQTtBQUNBO0FBQUE7QUFHQTs7OztBQXBGQTtBQUNBO0FBc0ZBIiwic291cmNlUm9vdCI6IiJ9\n//# sourceURL=webpack-internal:///./client/components/app.js\n");
```
Comment (therobinkim) [inline public/bundle.js]: You should never commit the contents of `bundle.js`; I suggest removing it from your repo entirely!

File: client/components/buttons.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import styles from './styles.css';
```
Comment (therobinkim) [inline client/components/buttons.js]: You can have multiple css files (eg, even one for each React Component).

File: client/components/productOption.js [hunk 4]
```diff
@@ -20,15 +21,17 @@ class ProductOption extends React.Component {
...
-              </option>
-            )
+        <div className={styles.optionLabel}>
+          <label className={styles.optionText}>{this.props.optionName}</label>
+        </div>
+        <select className={styles.optionSelect} value={this.state.optionSelection} onChange={this.handleDropDownSelection} >
+        <option value={''}>Select an option</option>
+        {this.props.choices.map((choice, idx) => {
+          return (
+            <option value={`${choice.choice} $${choice.adjustedPrice || ''}`} key={idx}>
```
Comment (therobinkim) [inline client/components/productOption.js]: I'm not 100% on what `choice.choice` is here, but your key can be `choice.choice` (or whatever other unique identifier); it'd be a better choice than `idx`.

**Comment by therobinkim** [comment]
Is `this.setState` a Promise that you can `await`? I saw this a few times in your code base but am unfamiliar w/ such usage (other than the fact that you can put `await` in front of anything, but doing so in front of a non-Promise doesn't really make sense).

**Comment by mflinn92** [comment]
> Is `this.setState` a Promise that you can `await`? I saw this a few times in your code base but am unfamiliar w/ such usage (other than the fact that you can put `await` in front of anything, but doing so in front of a non-Promise doesn't really make sense).

I think I will make an office hours appointment to talk about this in more detail. In these functions I am updating the local state of a component. And then I am using the local state to update the state of a parent component. This no longer works if I remove the await. I'm interested in this issue but may have to do some more digging to figure out exactly what is going on.

**Pull request merged**